lib/miq_automation_engine/models/miq_ae_datastore/xml_import.rb

```diff
@@ -155,8 +155,8 @@ def self.load_xml(f, domain_name = nil)
     end
 
     def self.create_domain(domain)
-      ns = MiqAeNamespace.find_by_fqname(domain)
-      MiqAeNamespace.create!(:name => domain, :enabled => true, :priority => 100) unless ns
+      ns = MiqAeDomain.where(:name => domain).first
+      MiqAeDomain.create!(:name => domain, :enabled => true, :priority => 100, :tenant => Tenant.seed) unless ns
```

Comment:
@gmcculloug good idea. I think that would read:

note: Since this is part of the main code base, so using `Tenant.root_tenant` is preferred over seed. (sorry if I had you change before)`

```ruby
def self.create_domain(domain)
  MiqAeDomain.create_with(:enabled => true, :priority => 100, :tenant => Tenant.root_tenant).find_or_create_by!(:name => domain)
```